docs/local-setup/readme.md

```diff
@@ -1,30 +1,35 @@
-
 ## Local Setup
 
 ### Step 1 : Pre Requisities
+
 - [Rust Basics](https://doc.rust-lang.org/book/)
 - [Cargo (Rust package manager)](https://doc.rust-lang.org/cargo/getting-started/installation.html)
 - [Docker (For containerization)](https://docs.docker.com/get-docker/)
 - [MongoDB Compass (For Visualising DB with GUI)](https://www.mongodb.com/try/download/compass)
 
-
 ### Step 2: Environment Variables
 
 To run this project, you will need to add the following environment variables to your .env file. Have a look at the `.env.example` file to see all the keys needed.
 
+You can generate `SERVER_KEK` by running the command below from the root of your project. ( Make sure you have cargo installed )
 
-You can generate `SERVER_KEK` by running the command below from the root of your project. ( Make sure you have cargo installed
 ```
 cargo run --bin create_kek
 
 ```
 
-
 SMTP servers require authentication to ensure that only authorized users can send emails. For generating `EMAIL_PASSWORD`, Visit this [link](https://support.google.com/mail/thread/205453566/how-to-generate-an-app-password?hl=en).
 `SMTP_DOMAIN = smtp.gmail.com` as we are using GMAIL as a Mail Provider.
 
+You have to add a random Access Key `X_API_KEY` for secure authentication of the endpoints.
+Alternatively you can use this command to generate a random `API KEY`
+
+```shell
+gpg --gen-random --armor 1 18
```

Comment:
Use open SSL to generate a secret. 

maybe `openssl rand -base64 32` as we are using open SSL for all of these. @mukeshkuiry

Reply:
Changed